.github/workflows/perlcritic.yml

```diff
@@ -12,10 +12,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-
-    - name: "Run Perl::Critic" 
-      uses: Difegue/action-perlcritic@2.1.0
```

Comment:
What is the motivation for not using the GitHub action https://github.com/Difegue/action-perlcritic ?

Reply:
It didn't seem to be working, it would pass my code when it has violations in it.